src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -197,9 +195,9 @@ protected void reloadConfig() {
     }
     try {
       log.info(String.format("Reloading hdfs-site.xml from %s", configUri));
-      String cfgCmd[] = new String[]{"sh", "-c",
-          String.format("curl -o hdfs-site.xml %s ; cp hdfs-site.xml etc/hadoop/", configUri)};
-      Process process = Runtime.getRuntime().exec(cfgCmd);
+      Process process = new ProcessBuilder("sh", "-c",
```

Comment:
Instead of method chaining, it is better java style to create a separate process builder object then call start on a separate line.